code/lib/shirts.js

```diff
@@ -12,7 +12,7 @@ module.exports = [
   },
   {
     $id: 2,
-    title: "Polo Shirt",
+    title: "Polo Shirts",
```

Comment:
what if the user wants to buy just 1 polo shirt? same comment for every change below in the file

Reply:
this title is shown in a card with a separate area that shows quantity.
There were already pluralized titles before I made this change, and I think it looks fine.

Reply:
ok maybe this doesn't look great. I'll have to change the training examples instead.

Reply:
actually I'll just add "s" for matching, in the js